docs/docs.json

```diff
@@ -482,5 +482,282 @@
 				}
 			]
 		}
-	]
+	],
+	"core-attribute": [
```

Comment:
I'm not sure about adding these directly to the JSON file of all other docs. It means we are downloading more data than needed, and we are not gaining anything from it, since we are not using them in any way. I'd (weakly) suggest using a separate file for this (index.json or something). Do note that there is precedent: The functions are also stored in a separate file.